### ISSUE TITLE
Regexp mask array in v-mask.2.2.0 version.

### ISSUE THREAD

**Issue by moveax3** [probil/v-mask#481]
Hello! Very thank you for your work!
I did try use regexp array in the new version:

In my model:
`car_number_mask: [/[АВЕКМНОРСТУХ]/, /[1-9]/, /[1-9]/, /[1-9]/, /[АВЕКМНОРСТУХ]/, /[АВЕКМНОРСТУХ]/, ' ', /[1-9]/, /[1-9]/, /[1-9]?/, /[A-Z]/, /[A-Z]/, /[A-Z]/],`

In my form:
`<b-form-input 
                    v-model="edit_form.truck.number"
                    v-mask="car_number_mask"
                    >`

But when I open my form, I see in input field:

> /\[АВЕКМНОРСТУХ\]/,/[1-9]/,/[1-9]/,/[1-9]/,/[АВЕКМНОРСТУХ]/,/[АВЕКМНОРСТУХ]/, ,/[1-9]/,/[1-9]/,/[1-9]

The browser console is clear, with no errors.
Please advise what I make wrong.

**Comment by davismcphee** [comment]
You're right, it's not working. It looks like the `dist` directory wasn't updated with the latest release, so I'm assuming it didn't get built.

This was my test case that should have worked:
`<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
  </head>
  <body>
    <div id="app"></div>
    <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/v-mask/dist/v-mask.min.js"></script>
    <script>
      Vue.use(VueMask.VueMaskPlugin);

      const myMaskedInput = new Vue({
        el: "#app",
        data() {
          return {
            car_number_mask: "###-###-###",
            value: "",
          };
        },
        template: `
        <div>
            <input v-mask="car_number_mask" v-model="value" />
            <div>{{value}}</div>
        </div>`,
      });
    </script>
  </body>
</html>`

**Comment by moveax3** [comment]
>But when I open my form, I see in the input field:

Sorry, this is the wrong information. 
With correct 2.2.0 version files has other problem with an array of regexp:

TypeError: e.split is not a function
at h (v-mask.2.2.0.min.js:formatted:197)
    at P (v-mask.2.2.0.min.js:formatted:262)
    at bind (v-mask.2.2.0.min.js:formatted:268)
    at callHook$1 (vue.js:6667)
    at _update (vue.js:6588)
    at Array.updateDirectives (vue.js:6569)
    at invokeCreateHooks (vue.js:6058)
    at initComponent (vue.js:5991)
    at createComponent (vue.js:5974)
    at createElm (vue.js:5914)

**Comment by probil** [comment]
> You're right, it's not working. It looks like the `dist` directory wasn't updated with the latest release, so I'm assuming it didn't get built.

In version v2.2.0 I've migrated the repo to semantic release. Probably there is some issue with build. Will check and re-publish soon. @moveax3 thanks for the issue!

**Comment by probil** [comment]
Hey @moveax3 
Please confirm whether issue is resolved in `v2.2.1`. 

Also I'd like to learn more about this issue:
```
TypeError: e.split is not a function
at h (v-mask.2.2.0.min.js:formatted:197)
at P (v-mask.2.2.0.min.js:formatted:262)
at bind (v-mask.2.2.0.min.js:formatted:268)
at callHook$1 (vue.js:6667)
at _update (vue.js:6588)
at Array.updateDirectives (vue.js:6569)
at invokeCreateHooks (vue.js:6058)
at initComponent (vue.js:5991)
at createComponent (vue.js:5974)
at createElm (vue.js:5914)
```
Can you try it with v2.2.1? In case of error please provide code sample to reproduce the issue as well as your browser (with version) and operating system

**Comment by moveax3** [comment]
Hey @probil 
Very thanks for your contribution!
I did try version 2.2.1, but has the same error:

> vue.min.js:6 TypeError: stringMask.split is not a function
>     at stringMaskToRegExpMask (v-mask.js:302)
>     at updateMask (v-mask.js:401)
>     at bind (v-mask.js:409)
>     at fr (vue.min.js:6)
>     at vue.min.js:6
>     at Array.sr (vue.min.js:6)
>     at y (vue.min.js:6)
>     at f (vue.min.js:6)
>     at h (vue.min.js:6)
>     at f (vue.min.js:6)

I did use the same code as @davismcphee (https://github.com/probil/v-mask/issues/481#issuecomment-633733177).
I attach to this message archive with an example.
[tmp.zip](https://github.com/probil/v-mask/files/4681768/tmp.zip)


Trying in browsers: 
SafaryVersion 13.1 (15609.1.20.111.8)
Google Chrome Version 81.0.4044.138 (Official Build) (64-bit)
Chromium Version 79.0.3917.0 (Developer Build) (64-bit)
in OsX 10.15.4 (19E287)
